misc/msvs/vcxproj.template

```diff
@@ -10,16 +10,18 @@
     <VCProjectUpgraderObjectName>NoUpgrade</VCProjectUpgraderObjectName>
   </PropertyGroup>
   %%PROPERTIES%%
-  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
+  <PropertyGroup Label="Configuration">
+    <PlatformToolset>%%PlatformToolset%%</PlatformToolset>
```

Comment:
You can't do this here, the platform toolset is a platform-specific setting, not a project-wide setting. I understand that Rider only uses clang, but in VS you can use any number of platform toolsets, and it's the platform that defines that setting for a specific build target configuration.

Reply:
Thank you for the information.
I will try to move this to the platform-specific props then.
Just to clarify, Rider also supports all the toolsets.